<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsPerRoomForm</class>
 <widget class="QWidget" name="OptionsPerRoomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>settingLabel</zorder>
   <zorder>back</zorder>
  </widget>
  <widget class="QWidget" name="frsiCustomContainer" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>195</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QPushButton" name="lower">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string notr="true">-</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="frsi">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.670000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="raise">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string notr="true">+</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="optionsBuildingContainer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>300</width>
     <height>145</height>
    </rect>
   </property>
   <property name="title">
    <string>Building type</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="2" column="0" colspan="2">
     <widget class="QWidget" name="substrateTypes" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="substrateTypeLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="substrateOne">
          <property name="text">
           <string>Substrate I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="substrateTwo">
          <property name="text">
           <string>Substrate II</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="buildingTypes" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="buildingTypeLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="newBuilding">
          <property name="text">
           <string>New building</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oldBuilding">
          <property name="text">
           <string>Old building</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="useCustom">
      <property name="text">
       <string>Custom building type</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>raise</sender>
   <signal>clicked()</signal>
   <receiver>frsi</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lower</sender>
   <signal>clicked()</signal>
   <receiver>frsi</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
